### ISSUE TITLE
How "not working" is this?

### ISSUE THREAD

**Issue by martyzz1** [luzfcb/cookiecutter-django-oauth#1]
Your setup looks like one which aligns with a new project I'm working on. I was wondering if you could spare the time to give a small braindump of whats not working so I can make a decision as to whether to fork and contribute, or roll my own...

**Comment by luzfcb** [comment]
I started doing integrations with [django-oauth-toolkit] (https://github.com/evonove/django-oauth-toolkit) and [django-rest-framework] (https://github.com/tomchristie/ django-rest-framework) to create an authentication server and authorization for an Android application I was developing. All I did was follow the documentation for both. https://github.com/luzfcb/cookiecutter-django-oauth/compare/03c34c547a49ab4f098ffb4bb359e26bd021e79e...master is only the changes was i made. After that, I did not have free time to do any testing. So marked as "not working" because I do not really know if it works.

https://www.djangopackages.com/grids/g/oauth-servers/

that's all I can do to help you.
